plugins/base/src/main/java/edu/wpi/first/shuffleboard/plugin/base/widget/RelayWidget.java

```diff
@@ -0,0 +1,101 @@
+package edu.wpi.first.shuffleboard.plugin.base.widget;
+
+import edu.wpi.first.shuffleboard.api.widget.Description;
+import edu.wpi.first.shuffleboard.api.widget.ParametrizedController;
+import edu.wpi.first.shuffleboard.api.widget.SimpleAnnotatedWidget;
+import javafx.fxml.FXML;
+import javafx.scene.layout.Pane;
+
+@Description(name = "Relay Widget", dataTypes = String.class)
+@ParametrizedController("RelayWidget.fxml")
+public class RelayWidget extends SimpleAnnotatedWidget<String> {
```

Comment:
We should have a data type for relays (see the `...plugin.base.data` package). That allows us to use this widget to autopopulate relays